module3/homework/inner_product/test.cpp

```diff
@@ -1,22 +1,38 @@
+#include <math.h>
 #include <vector>
 
 #include "arithmeticAverage.hpp"
 #include "gtest/gtest.h"
 
 bool cmp(double first, double second, double epsilon = 0.5) {
-    return (fabs(first - second) < epsilon);
+    return (std::fabs(first - second) < epsilon);
 }
 
 TEST(arithmeticAverageTest, ShouldCalculateArithmeticAverage) {
     std::vector<int> first{-5, 4, 2, -5, 2, 4, 6, 7};
     std::vector<int> second{3, 5, -2, 6, 7, -3, 6, 9};
     EXPECT_TRUE(cmp(ArithmeticAverage(first, second), 2.875));
-    // Write more test case
+    
+    first = {1, 2, 3, 4};
+    second= {1, 2, 3, 4};
+    ASSERT_EQ(ArithmeticAverage(first, second), 2.5);
+    EXPECT_TRUE(cmp(ArithmeticAverage(first, second), 2.875));
 }
 
```

Comment:
Good that you put there some news test cases! What do you think about testing vectors of different size? For example:
first = {1, 2, 3, 4, 5};
Second = {1};